test/trento_web/controllers/cluster_controller_test.exs

```diff
@@ -20,6 +24,66 @@ defmodule TrentoWeb.ClusterControllerTest do
     end
   end
 
+  describe "runner_callback" do
+    test "should return 400 when the request is invalid", %{conn: conn} do
```

Comment:
It is strange to have tests on failed cases and not in the happy path hehe
What it is what it is